installer/ubuntu/bootstrap

```diff
@@ -11,12 +11,12 @@
 # Grab the latest release of debootstrap
 echo 'Downloading latest debootstrap...' 1>&2
 d='https://anonscm.debian.org/gitweb/?p=d-i/debootstrap.git;a=snapshot;h=HEAD;sf=tgz'
-if ! wget -O- --no-verbose --timeout=60 -t2 "$d"  \
+if ! curl --connect-timeout 60 --retry 2 "$d"  \
         | tar -C "$tmp" --strip-components=1 -zx 2>/dev/null; then
     echo 'Download from Debian gitweb failed. Trying latest release...' 1>&2
     d='http://httpredir.debian.org/debian/pool/main/d/debootstrap/'
     d_api='https://sources.debian.net/api/src/debootstrap/'
-    f="$(wget -O- --no-verbose --timeout=60 -t2 "$d_api" \
+    f="$(curl --connect-timeout 60 --retry 2 --output - "$d_api" \
```

Comment:
the rest of them, too...

Reply:
unless there's a good reason otherwise, we can probably make all of our curl invocations look like the one in installer/crouton (minus the max-time bit)

`curl -# -L --connect-timeout 60 --retry 2 "$url"`

Reply:
I apologize if this is off base but is it only `bootstrap` needing the 'wget --> curl' replacement or are the others in need of it too -
```
chronos@localhost ~/Downloads $ find crouton-master -type f -exec grep -Hn 'wget ' {} +
crouton-master/installer/ubuntu/bootstrap:14:if ! wget -O- --no-verbose --timeout=60 -t2 "$d"  \
crouton-master/installer/ubuntu/bootstrap:19:    f="$(wget -O- --no-verbose --timeout=60 -t2 "$d_api" \
crouton-master/installer/ubuntu/bootstrap:29:    if ! wget -O- --no-verbose --timeout=60 -t2 "$d$f" \
crouton-master/installer/ubuntu/prepare:187:        local xvers="`wget -O- "$url" -nv \
crouton-master/installer/ubuntu/prepare:205:        wget "$pkgurl" -O"$deb"
crouton-master/targets/chrome-common:44:    wget "$url" -O "$tmp"
crouton-master/targets/audio:47:( wget -O "$archive" "$urlbase/$ADHD_HEAD.tar.gz" 2>&1 \
crouton-master/targets/audio:53:        wget -O "$archive" "$urlbase/$ADHD_HEAD.tar.gz"
crouton-master/targets/audio:55:        # wget already printed an explicit error
crouton-master/targets/core:113:install --minimal sudo wget ca-certificates apt-transport-https
crouton-master/targets/xiwi:60:wget $superinsecure -O "$DUMMYBUILDTMP/dummy.tar.gz" \
crouton-master/test/daemon.sh:254:wget "$TESTINGSSHKEYURL?format=TEXT" -O- | base64 -d > "$SSHKEY"
crouton-master/test/daemon.sh:273:    (wget -qO- "$QUEUEURL" && echo) | tail -n"+$lastline" \
crouton-master/chroot-bin/croutonversion:106:    if ! wget "$CROUTONURL" -O "$tmpdir/$INSTALLER" 2>"$tmpdir/log"; then
crouton-master/chroot-bin/croutonversion:130:    if ! wget "$CHANGESURL$BRANCH" -O/dev/full 2>&1 | grep -q '404 Not Found'; then
crouton-master/chroot-bin/croutonversion:132:    elif ! wget "${CHANGESURL}master" -O/dev/full 2>&1 | grep -q '404 Not Found'; then
```

~Dennis

Reply:
Anything that runs outside the chroot needs to be updated, at a minimum.  It would be good to be consistent and switch to curl everywhere.

I think this PR addresses all the instances that run outside of the chroot, but if you see one that isn't handled, please point it out.

Reply:
I get it now, sorry.
Maybe I'll do the others in a 'pie' at some point.

Thanx,
~Dennis

Reply:
No need to apologize; it's good to have an extra set of eyes on this stuff.

Reply:
Added `-# -L` to the rest of the curl invocations, except the one where we replace `wgetprogress` (just added `-L`)